### DISCUSSION TITLE
How do I know which propellors are CW and CCW

### DISCUSSION THREAD

**Discussion by akshaysunkara** [bitcraze/discussions#1428] [Q&A]
Hey guys, I recently got these propellors and needed help classifying whether they were Cw or CCW:

https://store.bitcraze.io/products/thrust-upgrade-bundle-for-crazyflie-2-x

This is a picture of the propellors:

![Untitled](https://github.com/bitcraze/discussions/assets/171630750/9e928408-08f1-4202-ad43-e9f2f3491169)
![Untitled](https://github.com/bitcraze/discussions/assets/171630750/d210f7a1-7d12-48fe-8c9f-8c0ce907658f)

**Comment by akshaysunkara** [comment]
Is it true that the ones that have R at the end (2.X...R) are clock-wise?

**Comment by gemenerik** [comment]
This may not be the easiest way, but what I usually do is identify what is the top side of the propeller and make sure they are upright. Then the rotational direction is towards the positive angle of attack of the blades. That is, the positive angle of attack side will turn into the air, generating lift upwards.
